Poke/App.tsx

```diff
@@ -15,9 +18,16 @@ const App = () => {
           <View style={styles.body}>
             <Text style={styles.simple}>Bem vindo(a) à Taqtile!</Text>
             
-            <EmailInput />
-            <SenhaInput />
-            <SubmitButton />
+            <EmailInput text={email} onTextChange={(text) => {
+              setEmail(text)
+            }}/>
```

Comment:
Dá pra passar só a referência da função:

```suggestion
            <EmailInput text={email} onTextChange={setEmail} />
```